File: Level2 - 다음 큰 숫자.py
def solution(n):
    i = 1
    n_b = format(n, 'b')
    n_b_cnt = str(n_b).count('1')

    while True:
        temp = format(n+i, 'b')
        temp_cnt = str(temp).count('1')
        if n_b_cnt == temp_cnt:
            return n+i
        i+=1

File: Level2 - 다음 큰 숫자.js
function solution(n) {
  const n_cnt = Array.from(n.toString(2)).filter((v) => v === "1").length;

  for (let i = n + 1; ; i++) {
    const temp = Array.from(i.toString(2)).filter((v) => v === "1").length;
    if (temp === n_cnt) return i;
  }
}
